separatemodels/src/test/java/pl/altkom/asc/lab/cqrs/intro/separatemodels/readmodel/PolicyInfoDtoFinderTest.java

```diff
@@ -0,0 +1,139 @@
+package pl.altkom.asc.lab.cqrs.intro.separatemodels.readmodel;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.SeparateModelsApplication;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.domain.Policy;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.CarsTestDataBuilder;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.PersonsTestDataBuilder;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.PoliciesTestDataBuilder;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = SeparateModelsApplication.class)
+public class PolicyInfoDtoFinderTest {
+
+    private static final String POLICY_NUMBER_POL_0001 = "POL0001";
+    private static final String POLICY_NUMBER_POL_0002 = "POL0002";
+    @Autowired
+    PolicyInfoDtoProjection policyInfoDtoProjection;
+
+    @Autowired
+    PolicyInfoDtoFinder policyInfoDtoFinder;
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    @Before
+    public void cleanTable() {
```

Comment:
bad naming - this method does more then cleaning the table, it also prepares the test data